api_docs/construct-narrow.md

```diff
@@ -51,7 +51,13 @@ important optimization when fetching messages in certain cases (e.g.
 when [adding the `read` flag to a user's personal
 messages](/api/update-message-flags-for-narrow)).
 
-**Changes**: In Zulip 9.0 (feature level 250), support was added for
+**Changes**: In Zulip 9.0 (feature level 263), narrows gained support for a new
+filter - `with`, taking a message ID as operand. If the message corresponding to
```

Comment:
Maybe `/filter/operator`, since it does more than just filtering?

Reply:
updated!